### DISCUSSION TITLE
Can React Hook Form Submit Despite Validation Errors?

### DISCUSSION THREAD

**Discussion by jmarc101** [react-hook-form/react-hook-form#11761] [Q&A]
I'm currently working on a project where I'm using React Hook Form for form validation. However, I'm facing a scenario where I need the form to submit even if there are validation errors. The reason behind this is that my validation involves checking against an external API, and the validity of certain fields might change asynchronously regardless of user input.

Is there a way to configure React Hook Form to allow form submission despite validation errors? I understand the importance of validation, but in this particular case, I need the flexibility to submit the form regardless of the validation status. Additionally, I'm open to making an open-source contribution if implementing such a feature aligns with the project's goals and guidelines.

Any suggestions or alternatives would be greatly appreciated! Thank you in advance for your help!

**Comment by Babailan** [comment]
I'm also looking for the exact same thing. any update?
